src/main/java/uk/gov/pay/ledger/payout/entity/PayoutEntity.java

```diff
@@ -21,13 +21,14 @@
     private String gatewayPayoutId;
     private Long amount;
     @JsonIgnore
-    private ZonedDateTime createdDate;
     @JsonSerialize(using = MicrosecondPrecisionDateTimeSerializer.class)
+    private ZonedDateTime createdDate;
```

Comment:
This looks like it moved created date under the serializer rather than being under `JsonIgnore`, this would change the behaviour of the class

Reply:
Oh no, the same field is being ignored, just looks like it's got a serialise rule now too which I think applies to the field below